### DISCUSSION TITLE
maa被defender报毒

### DISCUSSION THREAD

**Discussion by Leebais** [MaaAssistantArknights/MaaAssistantArknights#7576] [General]
Maa.exe被Windows defender报毒。![Screenshot_20231214-115431_Microsoft 远程桌面.png](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/29061618/e8960887-a02f-4c0b-9b94-79cde866c974)

**Comment by ABA2396** [comment]
#6662

**Comment by lanterner3054** [comment]
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/77407504/23ac1e79-0a03-43e1-9820-e91a366626ca)
我这边defender也报毒

**Comment by lanterner3054** [comment]
都是今天刚刚更新之后出的问题
